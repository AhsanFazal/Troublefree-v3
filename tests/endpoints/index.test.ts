describe("endpoints", () => {
  const filePath = "../../src/endpoints"
  const excluded = ["index.ts", "Base.ts"]
  it("should export all endpoints", () => {
    const fs = require("fs")
    const path = require("path")
    const files = fs.readdirSync(path.resolve(__dirname, filePath))
    const endpoints = files
      .filter((file: string) => !excluded.includes(file))
      .map((file: string) => file.replace(".ts", ""))
    expect(endpoints).toMatchSnapshot()
  })
})
