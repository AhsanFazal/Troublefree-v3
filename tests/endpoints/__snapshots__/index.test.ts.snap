// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`endpoints should export all endpoints 1`] = `
[
  "Articles",
]
`;
